data/sdo-creativework-examples.txt

```diff
@@ -13,10 +13,10 @@ MICRODATA:
 <meta itemprop="sameAs" content="https://en.wikipedia.org/wiki/The_Church_at_Auvers" />
 <span itemprop="name">The Church at Auvers</span>
 by <div itemprop="creator" itemscope itemtype="http://schema.org/Person">
-<span itemprop="name">Vincent van Gogh</span>
-</div>,
+  <span itemprop="name">Vincent van Gogh</span>
+</div>
 depicts a church in <div itemprop="contentLocation" itemscope itemtype="http://schema.org/AdministrativeArea">
-<span itemprop="name">Auvers-sur-Oise</span>,
+  <span itemprop="name">Auvers-sur-Oise</span>,
 </div>
```

Comment:
I'm seeing a couple of trivial edits to the "Church at Auvers" markup example that don't seem at all relevant to this PR #1615. Could you either retract them, or also fix the 'creaetd' in the same PR?